### PR TITLE
Adding security aspects of an image at: kubernetes.io/docs/concepts/c…

### PR DESCRIPTION
**Pull request by Biswajit-Mohapatra2**
Fixed:  #40530

Adding image security points to kubernetes documentation.

Changes at:  https://kubernetes.io/docs/concepts/containers/images/

### CODE REVIEW

**Review by dipesh-rawat — COMMENTED** [review]
Thanks for your contribution!
There is a small basic list of guidance around Images in the existing [Security Checklist](https://kubernetes.io/docs/concepts/security/security-checklist/#images) page.

**Comment by tengqm** [comment]
> Thanks for your contribution! There is a small basic list of guidance around Images in the existing [Security Checklist](https://kubernetes.io/docs/concepts/security/security-checklist/#images) page.

Good suggestion. Maybe we should try consolidate security related sections into that checklist, with links to that list added to the images page.

File: content/en/docs/concepts/containers/images.md [hunk 1]
```diff
@@ -416,6 +416,20 @@ common use cases and suggested solutions.
...
+
+Some common practices for image security include:
+
+- Regularly scanning container images for vulnerabilities and addressing any issues that are identified.
+- Verifying the authenticity and integrity of container images by using cryptographic signatures.
+- Implementing access controls to ensure that only authorized users have permission to build, modify, or deploy container images.
+- Ensuring that all dependencies used by container images are also secure and up-to-date.
+- Restricting the use of container images to trusted sources and avoiding the use of untrusted images or repositories.
+
+By implementing these measures and maintaining a robust security posture, organizations can help to minimize the risk of security incidents and data breaches that may result from using insecure images.
```
Comment (sftim) [inline content/en/docs/concepts/containers/images.md:431]: My opinion: it's better to put this inside https://kubernetes.io/docs/concepts/security/ and then change the **Image** concept to link to the new page.

That way, the image concept stays concise, and people who know that security is important can clearly see where to read on.

Comment (Biswajit-Mohapatra2) [reply content/en/docs/concepts/containers/images.md:431]: > then change the **Image** concept to link to the new page / new text.

@sftim : thank you for your suggestion, can you please explain the above line! i can't understand that line.

**Pull request closed without merging**